ci/README.md

```diff
@@ -6,6 +6,20 @@ Many of these scripts contain more detailed documentation and options in comment
 
 Any file and directory added into this tree should be documented here.
 
+## Publishing a release
+
+1. Change the version of code-server in `package.json` and push this commit.
+1. CI will run and generate an NPM package and release packages that you can download
+   as artifacts on Github Actions.
+1. Create a new draft release with the built release packages.
+1. Run some basic sanity tests on one of the released packages.
+1. Publish.
+1. Download the built npm package and publish it.
+1. Place the debian releases into `./release-packages` and then push the docker
```

Comment:
I don't know if this is better or worse but my strat was to pull the versioned Docker image, tag it latest, then push it back up.

Reply:
Actually I see in your other PR the image is pushed after a release is created so I suppose we could also just push latest? Since at that point we'd have verified the release is good to go.

Reply:
Yea will add.